v2.0/getting-started/kubernetes/installation/aws.md

```diff
@@ -278,23 +278,32 @@ Verify that you can access the Kubernetes API.  The following command should ret
 ```shell
 ./kubectl get nodes
 ```
-### 4.2 Configure Outbound NAT
+
+## 5. Install Addons
+
+{% include {{page.version}}/install-k8s-addons.md %}
```

Comment:
install-k8s-addons ends with "Next Steps" which send you off to the k8s network policy guide. Doesn't make sense to then have \#6 after it